server/plugin.go

```diff
@@ -49,21 +51,36 @@ func (p *Plugin) OnActivate() error {
 	teamSplit := split[0]
 	channelSplit := split[1]
 
-	team, err := p.API.GetTeamByName(teamSplit)
-	if err != nil {
-		return err
+	team, getTeamError := p.API.GetTeamByName(teamSplit)
```

Comment:
The naming convention for a `*model.AppError` return type is `appErr`.  Can you please `s/getTeamError/appErr/`?